.github/workflows/verible-ci.yml

```diff
@@ -51,23 +53,38 @@ jobs:
             FOLDER: /tmp/pages
             CLEAN: true
 
+
+   Matrix:
+      runs-on: ubuntu-latest
+      outputs:
+        matrix: ${{ steps.generate.outputs.matrix }}
+      steps:
+      - uses: actions/checkout@v2
+      - id: generate
+        run: |
+          matrix='['
+          for item in releasing/*/; do
+            matrix+='{"OS": "'"`echo $(basename $item) | cut -d- -f1`"'", "OS_VERSION": "'"`echo $(basename $item) | cut -d- -f2`"'"},'
+          done
+          matrix+=']'
+          echo "$matrix"
+          echo "::set-output name=matrix::$matrix"
+
+
    Release:
+      needs: Matrix
       runs-on: ubuntu-16.04
```

Comment:
We might want to consider not using such an ancient ubuntu here. Let's log an issue to loop back on that?

Reply:
See #626.